### ISSUE TITLE
movies are expanded full screen, cover the presentation, and never disappear

### ISSUE THREAD

**Issue by barak** [pdfpc/pdfpc#248]
I'm forwarding https://bugs.debian.org/867222

As soon as I upgraded to pdf-presenter-console/4.0.7-1, I began
experiencing a really really bad regression regarding movie playback.

Supposed to have a PDF presentation (created with LaTeX and Beamer)
that includes movies incorporated with the following LaTeX code:
```latex
  \newcommand{\includeavi}[2]{\movie[loop]{\includegraphics[#1]{figs/#2}}{#2.avi}}
  [...]
  \includeavi{width=0.73\textwidth}{mymovie}
```
When pdf-presenter-console/4.0.7-1 shows the presentation, something
awkward happens, as soon as the page incorporating the movie is reached:
the movie is expanded full screen and covers the actual presentation
page. The movie may be played, but there seems to be no way to make it
disappear again. Hitting [space] or [PageUp] or [PageDown] seems to
have no effect on the presentation screen. On the presenter console,
the preview of the next slide changes, hence I assume that the other
slides of the presentation are actually shown, but they are masked
by the expanded movie!

In conclusion there seems to be no way to correctly show a presentation
incorporating movies...

pdf-presenter-console/4.0.6-1 works as expected.

**Comment by phillipberndt** [comment]
Duplicate of #240. I'm working on a fix right now.
